dids/diddht/diddht.go

```diff
@@ -86,18 +50,22 @@ func (rec *dhtDIDRecord) DIDDocument(didURI string) *didcore.Document {
 
 			opts := []didcore.Purpose{}
 			for _, r := range relationships {
-				if o, ok := relationshipDNStoDID[r]; ok {
+				if o, ok := vmPurposeDNStoDID[r]; ok {
 					opts = append(opts, o)
```

Comment:
My tests are failing to compile because of this line. Is it just me?

> /Users/kendallw/Development/workspace/web5-go/dids/diddht/diddht.go:54:26: cannot use o (variable of type string) as didcore.Purpose value in argument to append

Reply:
```go
opts = append(opts, didcore.Purpose(o))
```

Reply:
hum... must have been some merge issue. I just saw my local branch diverged for some reason. my local didn't have `opts := []didcore.Purpose{}` it was using strings.